src/components/Loader.scss

```diff
@@ -0,0 +1,248 @@
+    #{$prefix}transform: translateY($su-default) scale(1,.9) rotate(45deg) ;
+  }
+  75% {
+    #{$prefix}transform: translateY($su-small) rotate(67.5deg);
+  }
+  100% {
+    #{$prefix}transform: translateY(0) rotate(90deg);
+  }
+}
+
+@keyframes jelloBox {
+  @include jelloBox('');
+}
+
+@-webkit-keyframes jelloBox {
+  @include jelloBox('-webkit-');
+}
+
+@mixin jelloShadow($prefix) {
+  50% {
+    #{$prefix}transform: scale(1.2,1);
+  }
+}
+
+@keyframes jelloShadow {
+  @include jelloShadow('');
+}
+
+@-webkit-keyframes jelloShadow {
+  @include jelloShadow('-webkit-');
+}
+
+// * 1.b Pulse Indicator
+
+.PulseAnimation {}
+
+.PulseAnimation-dot {
+  display: inline-block;
+  width: $su-small;
+  height: $su-small;
+  background-color: $cu-info--light;
+  border-radius: 100%;
+  -webkit-animation: PulseAnimation 1.5s infinite ease-in-out both;
+  animation: PulseAnimation 1.5s infinite ease-in-out both;
+}
+
+.PulseAnimation-dot:nth-child(1) {
+  -webkit-animation-delay: -500ms;
+  animation-delay: -500ms;
+}
+
+.PulseAnimation-dot:nth-child(2) {
+  -webkit-animation-delay: -250ms;
+  animation-delay: -250ms;
+}
+
+@mixin PulseAnimation($prefix) {
+  0%, 80%, 100% {
+    #{$prefix}transform: scale(0)
+  }
+  40% {
+    #{$prefix}transform: scale(1.0)
+  }
+}
+
+@keyframes PulseAnimation {
+ @include PulseAnimation('');
+}
+
+@-webkit-keyframes PulseAnimation {
+  @include PulseAnimation('-webkit-');
+}
+
+// * 1.c Spin Indicator
+
+.SpinAnimation {
+  display: inline-block;
+  width: $su-large;
+  height: $su-large;
+  border-radius: 50%;
+  vertical-align: middle;
+  margin: $su-xsmall;
+  border-top: solid $su-xxsmall $cu-info;
+  border-right: solid $su-xxsmall transparent;
+  box-shadow: 0 0 0 1px $cu-info--light;
+  -webkit-animation: spin 1s linear infinite;
+  animation: spin 1s linear infinite;
+}
+
+@mixin SpinAnimation($prefix) {
+  0% {
+    #{$prefix}transform: rotate(0deg);
+  }
+  50% {
+    #{$prefix}transform: rotate(180deg);
+  }
+  100% {
+    #{$prefix}transform: rotate(360deg);
+  }
+}
+
+@keyframes spin {
+  @include SpinAnimation('');
+}
+
+@-webkit-keyframes spin {
+  @include SpinAnimation('-webkit-');
+}
+
+// * 2 Page Loader
+
+.PageLoader {
+  width: 100%;
+  height: 125px;
+  text-align: center;
+  display: block;
+}
+
+.PageLoader-indicator {
+  position: relative;
+  display: inline-block;
+  height: $su-xlarge;
+  padding-left: $su-xlarge;
+  margin-bottom: $su-small;
+}
+
+.PageLoader-indicatorAnimation {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  margin: auto;
+}
+
+.PageLoader-indicatorText {
+  font-size: $tu-large-fontSize;
+  font-weight: $tu-bold-fontWeight;
+  line-height: $su-xlarge;
+  color: $cu-info;
+}
+
+.PageLoader-message {
+  display: block;
+  font-size: $tu-large-fontSize;
+  padding: $su-default 0;
+  color: $cu-info--light;
+}
+
+// * 2.a Page Loader Modifiers
+
+.PageLoader--centered {
+  position: fixed;
+  top: 50%;
+  left: 0;
+  margin-top: -63px;
```

Comment:
This might be able to get there without pixel values using `transform: translateY(-50%)`.